docs: Fix markdown formatting in README

jodh-intel opened the pull request:
Some of the formatting in the README was incorrect (verbatim text when
it should have been normal and vice versa).

Fixes #32.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

Commit:
docs: Fix markdown formatting in README

Some of the formatting in the README was incorrect (verbatim text when
it should have been normal and vice versa).

Fixes #32.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

grahamwhaley commented:
Whilst here - can I ask you to pick up another one. In the introduction, there seems to be a broken ref:
`See [Environment Variables](#Environment Variables).`
I think that may render if we go to [Environment Variables](#environment-variables) : `[Environment Variables](#environment-variables)`
Oh, and there seems to be multiple occurrences of that in the doc.

Commit:
docs: Fix links to "Environment Variables" section

All the links to the "Environment Variables" section were using the
wrong syntax.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

grahamwhaley commented:
thanks for the extra fix @jodh-intel
